rfcs/0001-bootstrapping-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: bootstrapping-rfc-process
+- Start Date: 09 February 2018
+- RFC PR: (leave this empty)
+- IPFS Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Begin using an RFC process. Initially use a structure similar to the process used by the maintainers of Rust lang, as described in https://github.com/rust-lang/rfcs
+
+# Motivation
+[motivation]: #motivation
+
+Establish a clear, repeatable process for people to propose ideas and make decisions in a way that
+1. Allows everyone to see what decisions have been made and how they were made
+2. Allows everyone to see what proposals have been proposed and comment on them or ssuggest changes
+3. Works for a globally distributed group of contributors to discuss and make decisions asynchronously
+4. Can be replicated/repeated in sibling projects or spinoff projects like libp2p, IPLD, etc.
```

Comment:
In the beginning, this process applies only for IPFS features or we'll use this repository for proposals for the rest of the projects as well?

Reply:
Later RFCs will spell out the org structure, which will decide which projects are affected by the RFCs in this repo. See #2, which proposes making a separate libp2p org. That org might have its own RFCs...

Reply:
oops - #3 is the libp2p RFC. #2 is the (very incomplete) proposal for org structure